qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -194,11 +195,12 @@ def save_account(
         """
         if url != QX_AUTH_URL:
             raise IBMQAccountCredentialsInvalidUrl(
-                'Invalid IBM Q Experience credentials found. ' + UPDATE_ACCOUNT_TEXT)
+                'The URL specified ({}) is not an IBM Quantum Experience authentication '
+                'URL. Valid authentication URL: {}.'.format(url, QX_AUTH_URL))
```

Comment:
In this case, the user trying to save an account to disk, with a specific URL. I thought it would be best to point them to the correct URL to use if they are using the incorrect one. But, there is the possibility of using an old url, in which case it's best to point them towards how to update their account. So, I'll switch it back! 👍